packages/web3-plugin-example/webpack.config.js

```diff
@@ -0,0 +1,20 @@
+/*
+This file is part of web3.js.
+
+web3.js is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+web3.js is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+const { getWebPackConfig } = require('../../webpack.base.config');
+
+module.exports = getWebPackConfig(__dirname, 'web3.min.js', 'Web3', 'src/web3.ts');
```

Comment:
```suggestion
module.exports = getWebPackConfig(__dirname, 'web3-plugin-example.min.js', 'web3-plugin-example', 'src/index.ts');
```

Reply:
Removed the file in [this](https://github.com/ChainSafe/web3.js/pull/5393/commits/9b4896e5cacc8e83b93999cc16492813389f38d5) commit, was an erroneous copy